src/Features/Core/Portable/CodeRefactorings/SyncNamespace/IChangeNamespaceService.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.LanguageServices;
+
+namespace Microsoft.CodeAnalysis.ChangeNamespace
+{
+    internal interface IChangeNamespaceService : ILanguageService
```

Comment:
Another way to think about this.  There are two types of ILanguageServices we want:
1. a way to support a feature that should work across VB/C#.  This type of service can return to things like SyntaxNodes, Compilations, SemanticModels, etc.
2. a way for any language to plug in if they think they have a reasonable way to support tthings.  This type of service works with things like Solutions, Documents, strings, and simple types we pass in/out (like outlining, find-refs, etc.)

Right now this interface is both kinds.  We could split it into two.  That way a language like F#/TS/JS could participate here by implementing hte latter if we wanted to ever expose a global command to 'synchronize namespace'.  Then, the C#/VB impl would be shared and would defer to an impl that implemented the former in order to simplify the C#/VB impl.